hackathon/models.py

```diff
@@ -112,15 +114,20 @@ class HackProject(models.Model):
     "scores" has been moved to HackProjectScore. See comments there."""
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
-    # Each model can only be created by one user: One To Many
-    created_by = models.ForeignKey(User,
-                                   on_delete=models.CASCADE,
-                                   related_name="hackprojects")
+    created_by = models.ForeignKey(
+        User,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="hackproject",
+        default="")
     display_name = models.CharField(default="", max_length=255)
-    description = models.TextField()
-    github_link = models.URLField(default="", max_length=255)
-    collab_link = models.URLField(default="", max_length=255)
+    description = models.TextField(max_length=500)
+    github_url = models.URLField(default="", max_length=255)
+    deployed_url = models.URLField(default="", max_length=255)
     submission_time = models.DateTimeField(auto_now_add=True)
+    speaker_name = models.CharField(default="", max_length=225)
+    share_permission = models.BooleanField(default=True)
```

Comment:
Do we need this here? Or more specifically can the person submitting the project legally agree to the sharing permissions for all team members? I would say we put something in the T&Cs when signing up that they need to confirm that any work that is produced as part of participating on this platform can be shared. @TravelTimN what do you think about this?

Reply:
Yeah @TravelTimN